x-pack/plugins/security_solution/public/management/components/response_action_file_download_link/response_action_file_download_link.tsx

```diff
@@ -65,16 +64,11 @@ export const ResponseActionFileDownloadLink = memo<ResponseActionFileDownloadLin
     const getTestId = useTestIdGenerator(dataTestSubj);
     const { canWriteFileOperations } = useUserPrivileges().endpointPrivileges;
 
-    // We don't need to call the file info API every time, especially if this component is used from the
-    // console, where the link is displayed within a short time. So we only do the API call if the
-    // action was completed more than 2 days ago.
-    const checkIfStillAvailable = useMemo(() => {
-      return (
-        action.isCompleted && action.wasSuccessful && moment().diff(action.completedAt, 'days') > 2
-      );
-    }, [action.completedAt, action.isCompleted, action.wasSuccessful]);
+    const shouldFetchFileInfo: boolean = useMemo(() => {
+      return action.isCompleted && action.wasSuccessful;
+    }, [action.isCompleted, action.wasSuccessful]);
```

Comment:
This change is unexpected to me, based on the PR's description. Is it intentional?

Reply:
Yes, it is intentional. We removed the additional condition to only fetch file info if the action was at least 2 day old. Now we fetch the file info as soon as the component is rendered and if the action is completed and successful.